tests/extensions/countries.sql

```diff
@@ -469,4 +469,3 @@ COPY capitals (id, name, country_id, geom) FROM stdin;
 --
 -- PostgreSQL database dump complete
 --
-
```

Comment:
Not our file, is it possible to avoid processing it?

Reply:
I did not know about this file being from an outside source. Thanks for bringing this to my attention.

I added a new .editorconfig in this directory to have an exclusion for this file. All the other files in the same directory are still affected by the root .editorconfig.